LiveBlip/Components/Pages/Welcome.razor

```diff
@@ -1,18 +1,88 @@
-﻿@page "/welcome"
+﻿
+@page "/welcome"
+@inject NavigationManager NavManager
 
 <PageTitle>Welcome</PageTitle>
 
-<div class="welcome-card">
-	<div class="page-heading">
-		<h1>Welcome to</h1>
-		<img src="images/logo.png" class="logo"
-	</div>
 
-	<div class="input-imei">
-		<input type="text" class="form-control-imei" placeholder="Please input tracker IMEI" />
-	</div>
+<MudPaper Class="mx-auto my-16 pa-6 mud-theme-dark border-solid border-2 mud-border-primary pa-4 " Height="230px" Width="50%" Elevation="22" Outlined="true" >
+
+
+    <MudContainer MaxWidth="MaxWidth.Small" Class="d-flex align-content-center flex-wrap flex-grow-1 gap-4 ">
+        <MudText Class=" pa-4 pl-14 " Color="Color.Secondary" Typo="Typo.h4"><b>Welcome to</b > </MudText>
+		<MudImage Class="ml-n9"  Src="images\logo.png" Width="250" Height="70"/>
+     </MudContainer>
+
+
+
+<MudForm  @ref="form" @bind-IsValid="@success" @bind-Errors="@errors" >
+     <MudContainer  Class=" ml-7 d-flex align-content-center flex-wrap flex-grow-1 gap-4" >
+
+          <MudTextField  @bind-Value="HelperText" Label="Please input tracker IMEI"  Variant="Variant.Outlined" Margin="Margin.Dense"   @ref="imeiField1" Validation="@(new Func<string, IEnumerable<string>>(ImeiValidation))" Required="true" RequiredError="Correct IMEI number is required!"/>
```

Comment:
I can guarantee you that there is a way to make that background behind the text match the rest of the text field